ior.spec

```
@@ -50,18 +50,16 @@ IOR-HPC
 
 %prep
 %autosetup -p1
-%if "%{?commit}" != ""
-# we most likely patched configure.ac
+# we patched configure.ac
 autoreconf
-%endif
 
 %build
 export CC=mpicc
 export CXX=mpicxx
 export FC=mpif90
 export F77=mpif77
-export CFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -fPIC"
-export CXXFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -fPIC"
+export CFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -fPIC -fPIE"
+export CXXFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -fPIC -fPIE"
```

Since EL >= 8 defaults to hardening their builds, we should gate these two lines on being SUSE only:

```suggestion
%if (0%{?suse_version} >= 1500)
export CFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -fPIC -fPIE"
export CXXFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing -fPIC -fPIE"
%endif
```

so that we don't interfere with EL doing the right thing on it's own.

---

Done